### PR TITLE
fix:repeat startup

### PR DESCRIPTION
**Pull request by songzhibin97**

### CODE REVIEW

File: holmes.go [hunk 1]
```diff
@@ -159,7 +160,9 @@ func (h *Holmes) startGCCycleLoop() {
 
 // Start starts the dump loop of holmes.
 func (h *Holmes) Start() {
-	atomic.StoreInt64(&h.stopped, 0)
+	if !atomic.CompareAndSwapInt64(&h.stopped, 1, 0) {
+		return
```
Comment (doujiang24) [inline holmes.go:164]: maybe it's better to add a long when restarting it again?

Comment (songzhibin97) [reply holmes.go:164]: I don't quite understand.
Add a restart?

Comment (doujiang24) [reply holmes.go:164]: add a log before return since start twice should be unexpected.
the log is just a warning for the caller.

Comment (songzhibin97) [reply holmes.go:164]: OK, I think holmes can be configured to start externally, and repeated booting may be expected.the output log is also correct.

**Review by doujiang24 — APPROVED** [review]

**Pull request merged**